fuel_cell/actuators.py

```diff
@@ -1,10 +1,47 @@
 """
+Controls a valve's actuator
 """
 
+import RPi.GPIO as GPIO
 
-class Actuator(object):
+
+class Actuator:
+    def __init__(self, output_pin, error_pin, closed_on_low):
+        GPIO.setmode(GPIO.BCM)
+
+        self.output_pin_ = output_pin
+        GPIO.setup(self.output_pin_, GPIO.OUT)
+        # NOTE: should we always ensure the actuator is in its default
+        # position? Does the following line garantee that?
+        GPIO.output(self.output_pin_, GPIO.LOW)
+        self.error_pin_ = error_pin
+        self.closed_on_low_ = closed_on_low
+    
     def open_valve(self):
-        pass
+        if self.closed_on_low_:
+            GPIO.output(self.output_pin_, GPIO.HIGH)
+        else:
+            GPIO.output(self.output_pin_, GPIO.LOW)
 
     def close_valve(self):
-        pass
+        if self.closed_on_low_:
+            GPIO.output(self.output_pin_, GPIO.LOW)
+        else:
+            GPIO.output(self.output_pin_, GPIO.HIGH)
+
+    def get_status(self):
+        return bool(GPIO.input(self.error_pin_))
+
+if __name__ == "__main__":
+    import time
+    # TODO: make sure the actuator doesn't move right after creating the object
+    actuator = Actuator(23, 5, True)
+    time.sleep(5)
```

Comment:
Sus

Reply:
On a testé avec Émile à l'attelier. Lorsque l'actuateur reçoit de la puissance (V in), i, effectue une petite séquence de calibration pour le contrôleur PID qui y est intégré. Ce mouvement de quelques degrés n'est pas suffisant pour ouvrir la valve et ne devrait pas s'effectuer lorsque le PI envoi des signaux d'ouverture/fermeture